tree/dataframe/src/RDataFrame.cxx

```diff
@@ -501,19 +501,15 @@ You see how we created one `double` variable for each thread in the pool, and la
 Friend trees are supported by RDataFrame.
 In order to deal with friend trees with RDataFrame, the user is required to build
 the tree and its friends and instantiate a RDataFrame with it.
-Two caveats are presents when using jitted `Define`s and `Filter`s:
-1) the only columns which can be used in the strings passed to the aforementioned transformations are the top level branches of the friend trees.
-2) the "friend columns" cannot be written with the notation involving a dot. For example, if a tree is created like this:
+One caveat is present when using jitted `Define`s and `Filter`s:
+The only columns which can be used in the strings passed to the aforementioned transformations are the top level branches of the friend trees.
```

Comment:
This is not true anymore, you can pass the same depth of branch names with friend trees as with the main tree. So, you can do "myFriend.myBranch.mySubBranch.myLeaf"

Reply:
ok, i'll adapt it.